dev/VSIX/ProjectTemplates/Desktop/CppWinRT/SingleProjectPackagedApp/readme.txt

```diff
@@ -3,23 +3,23 @@
 ========================================================================
 
 This project demonstrates how to get started writing WinUI3 apps directly
-with standard C++, using the Project Reunion and C++/WinRT packages and 
-XAML compiler support to generate implementation headers from interface 
-(IDL) files. These headers can then be used to implement the local 
+with standard C++, using the Windows App SDK and C++/WinRT packages and
+XAML compiler support to generate implementation headers from interface
+(IDL) files. These headers can then be used to implement the local
 Windows Runtime classes referenced in the app's XAML pages.
 
 Steps:
-1. Create an interface (IDL) file to define any local Windows Runtime 
+1. Create an interface (IDL) file to define any local Windows Runtime
     classes referenced in the app's XAML pages.
-2. Build the project once to generate implementation templates under 
-    the "Generated Files" folder, as well as skeleton class definitions 
-    under "Generated Files\sources".  
+2. Build the project once to generate implementation templates under
+    the "Generated Files" folder, as well as skeleton class definitions
+    under "Generated Files\sources".
 3. Use the skeleton class definitions for reference to implement your
     Windows Runtime classes.
 
 ========================================================================
-Learn more about Project Reunion here:
-https://docs.microsoft.com/en-us/windows/apps/project-reunion/
+Learn more about Windows App SDK here:
+https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/
```

Comment:
Wasn't your changes, but let's remove the `en-us` from the link while we're at it :)

```suggestion
https://docs.microsoft.com/windows/apps/windows-app-sdk/
```